### ISSUE TITLE
add tabindex=0 to treebase

### ISSUE THREAD

**Issue by pkra** [krautzource/aria-tree-walker#24]
Instead of expecting authors to do it, aria-tree-walker should add tabindex as progressive enhancement. Otherwise, if aria-tree-walker fails, broken trees will be left in the tabindex, which might cause confusion.
